pkg/cli/config.go

```diff
@@ -29,15 +29,15 @@ type HostConfig struct {
 	GCP GCPHostConfig
 }
 
-type Config struct {
+type FileConfig struct {
```

Comment:
Adding the `File` prefix seems  an implementation detail, where the config is pulled from. This struct represents the overall configuration of cvdremote before any command gets executed no matter how it's parameters are set.

Reply:
Agreed, but just `Config` sounds like the union of the file config and the flags.
Accepting suggestions for a different name.

Reply:
Done